src/main.h

```diff
@@ -15,6 +15,7 @@
 #include "timestampindex.h"
 #endif // ENABLE_ADDRESS_INDEXING
 
+#include "addressindex.h"
```

Comment:
I wonder if this include should go under a conditionally compiled section since it is used by the Address Indexing only.

```suggestion
#ifdef ENABLE_ADDRESS_INDEXING
#include "addressindex.h"
#endif // ENABLE_ADDRESS_INDEXING
```

Reply:
Done (`"addressindex.h"` include replaced by `AddressType` forward declaration)